This is not valid Perl, I don't really understand why it is here anyway.

SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: Artistic-1.0

--- dist-3.5-236/bin/perload	2021-04-21 19:01:35.803767498 +1000
+++ dist-3.5-236/bin/perload	2021-04-21 19:01:47.959850217 +1000
@@ -450,8 +450,6 @@
 :# This perl program uses dynamic loading [generated by perload]
 :#
 :
-:$ENV{LC_ALL} = 'C';
-:
 EOC
 	$init_emitted = 1;
 }
